doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -19,29 +19,28 @@
 
 ## Executive Summary
 
-The RC2 was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet industry driven quality assurance standards for conformance, verification, and validation. For the purpose of this chapter, infrastructure and cloud native network function (CNF) testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
+The RC2 was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet functional and performance requirements derived from RA2 and the CNTT Reference Model (RM). For the purpose of this chapter, cloud infrastructure (NFVI) and cloud native network function (CNF) testing will be specified to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of hardware and software delivery that is within the scope of CNTT. Conformance scope includes:
```

Comment:
Can you remove NFVI since we might be running RI on bare metal in which case there would not be virtualization?

Reply:
NFVI has been removed.